lib/mspec/guards/platform.rb

```diff
@@ -54,22 +54,34 @@ def self.wsl?
   end
 
   WORD_SIZE = 1.size * 8
+  deprecate_constant :WORD_SIZE
 
   POINTER_SIZE = begin
     require 'rbconfig/sizeof'
     RbConfig::SIZEOF["void*"] * 8
   rescue LoadError
-    WORD_SIZE
+    [0].pack('j').size
```

Comment:
This fallback and the one below were incorrect, fixed in https://github.com/ruby/mspec/commit/92ee9f7c8cdeb72f3b58c755cf671e2a27227e02

Reply:
Ah good catch.